.github/workflows/kuksa_databrokercli_build.yml

```
@@ -0,0 +1,154 @@
+          type=semver,pattern={{major}}.{{minor}}
+          type=semver,pattern={{major}}
+          
+    # only needed for runners without buildx setup, will be slow 
+    #- name: Set up QEMU
+    #  uses: docker/setup-qemu-action@v2
+      
+    # - name: Set up Docker Buildx
+    #  id: buildx
+    #  uses: docker/setup-buildx-action@v2
+
+    - name: Log in to the Container registry
+      if: needs.checkrights.outputs.have_secrets == 'true'
+      uses: docker/login-action@v2
+      with:
+          registry: ${{ needs.checkrights.outputs.registry }}
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+    - name: Build kuksa.val databroker CLI container and push to ghcr.io
+      id: ghcr-build
+      if: needs.checkrights.outputs.have_secrets == 'true'
+      uses: docker/build-push-action@v3
+      with:
+        platforms: |
+          linux/amd64
+          linux/arm64
+        file: ./kuksa_databroker/Dockerfile-cli
+        context: .
+        push: true
+        tags: |
+          ${{ steps.meta.outputs.tags }}
+          ttl.sh/kuksa.val/kuksa-databrokercli-${{github.sha}}:1h
+        labels: ${{ steps.meta.outputs.labels }}
+
+    - name: Build ephemereal kuksa.val databroker container and push to ttl.sh
+      if: needs.checkrights.outputs.have_secrets == 'false'
+      id: tmp-build
+      uses: docker/build-push-action@v3
+      with:
+        platforms: |
+          linux/amd64
+          linux/arm64
+        file: ./kuksa_databroker/Dockerfile-cli
+        context: .
+        push: true
+        tags: "ttl.sh/kuksa.val/kuksa-databrokercli-${{github.sha}}:1h"
+        labels: ${{ steps.meta.outputs.labels }}
+        
+    - name: Posting message
+      run: |
+        echo "## :hatching_chick: Moin!" >> $GITHUB_STEP_SUMMARY
+        echo "Something new is in the world" >> $GITHUB_STEP_SUMMARY 
+        echo -e "\nImages for testing temporarily available at \`ttl.sh/kuksa.val/kuksa-databrokercli-${{github.sha}}:1h\`" >> $GITHUB_STEP_SUMMARY
+        echo -e "\n\`\`\`\ndocker pull ttl.sh/kuksa.val/kuksa-databrokercli-${{github.sha}}:1h\n\`\`\`" >> $GITHUB_STEP_SUMMARY
+    
+    - name: Extracting AMD64 binaries
+      run: | 
+         docker rm cliamd || true
+         docker create --name cliamd --platform linux/amd64 ttl.sh/kuksa.val/kuksa-databrokercli-${{github.sha}}:1h
+         docker cp cliamd:/app ./  
+         docker rm cliamd
+         mv ./app ./databrokercli-amd64
+         
+    - name: Extracting ARM64 binaries
+      run: | 
+         docker rm cliarm || true
+         docker create --name cliarm --platform linux/arm64 ttl.sh/kuksa.val/kuksa-databrokercli-${{github.sha}}:1h
+         docker cp cliarm:/app ./  
+         docker rm cliarm
+         mv ./app ./databrokercli-arm64
+         
+    - name: Archive  AMD64 binaries
+      uses: actions/upload-artifact@v3
+      with:
+        name: databrokercli-amd64
+        path: ./databrokercli-amd64 
```

```suggestion
        path: ./databroker-cli-amd64.tar.gz
```
Use created tar instead. File mode (executable) is lost when using upload-artifact for zipping the directory.